pkgs/development/python-modules/hfst/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, pkgs
+, stdenv
+, buildPythonPackage
+, icu
+, swig
+}:
+
+buildPythonPackage {
+  pname = "hfst";
+  inherit (pkgs.hfst) version src;
+
+  sourceRoot = "source/python";
+
+  postPatch = ''
+    # omorfi-python looks for 'hfst' Python package
+    sed -i 's/libhfst_swig/hfst/' setup.py;
+  '';
+
+  nativeBuildInputs = [
+    swig
+  ];
+
+  buildInputs = [
+    icu
+    pkgs.hfst
+  ];
+
+  setupPyBuildFlags = [
+    "--inplace"
+  ];
+
+  meta = with lib; {
+    description = "Python bindings for HFST";
+    homepage = "https://github.com/hfst/hfst";
+    license = licenses.gpl3;
+    maintainers = with maintainers; [ lurkki ];
+    # Ofborg build error
+    broken = stdenv.isDarwin;
```

Comment:
```suggestion
```
Darwin build is OK now, but some tests failed, so `doCheck = !stdenv.isDarwin;` required as workaround.